packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1647,6 +1656,8 @@ abstract class DiagnosticsNode {
   TextTreeConfiguration get textTreeConfiguration {
     assert(style != null);
     switch (style) {
+      case DiagnosticsTreeStyle.none:
```

Comment:
now you will NPE inside the toStringDeep code unless you add extra checks there to short circuit.

Reply:
Changed that.